src/bp/core/services/ghost/disk-driver.ts

```diff
@@ -92,4 +97,25 @@ export default class DiskStorageDriver implements StorageDriver {
       return []
     }
   }
+
+  async discoverTrackableFolders(baseDir: string): Promise<string[]> {
+    try {
+      const allFiles = await this.directoryListing(baseDir, undefined, true)
+      const allDirectories = this._getBaseDirectories(allFiles)
+      const noghostFiles = allFiles.filter(x => path.basename(x).toLowerCase() === '.noghost')
+      const noghostDirectories = this._getBaseDirectories(noghostFiles)
```

Comment:
what if someone wants to sync some files in a dir ?

should we support both: 
```
somedir >
  .noghost
  file1.json
```
and 

```
somedir >
  file1.json
  file2.noghost.json
```

Reply:
maybe we can introduce a `.ghostrc` file with glob patterns to address this use case if we see need?